### PR TITLE
u3d: add page

### PR DESCRIPTION
**Pull request by 41407**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known): v1.2.3**

### CODE REVIEW

File: pages/common/u3d.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Open project from the current directory in correct Unity version.
+
+`u3d`
+
+- List installed versions of Unity.
+
+`u3d list`
+
+- List download-ready versions of Unity.
```
[the remark this answers is not in the document]
Comment (owenvoke) [reply pages/common/u3d.md:14]: I was just trying to differentiate between versions that are installed and versions that are available to download. 🤷🏻

File: pages/common/u3d.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`u3d available`
+
+- Download and install latest stable Unity version.
+
+`u3d install {{latest_stable}}`
```
[the remark this answers is not in the document]
Comment (41407) [reply pages/common/u3d.md:20]: Yes; `latest_stable` is a constant but user can specify a version like `2021.1.16f1` as well. I thought `latest_stable` might be a useful usage scenario but conveying the significance of the parameter is a bit difficult, any suggestions?

File: pages/common/u3d.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (41407) [reply pages/common/u3d.md:14]: @navarroaxel A version might have been deprecated, and therefore not available for downloading.

File: pages/common/u3d.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (navarroaxel) [reply pages/common/u3d.md:20]: I think this makes more clear that it's a valid value and not a placeholder
```suggestion
`u3d install latest_stable`
```

**Review by navarroaxel — APPROVED** [review]

File: pages/common/u3d.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (41407) [reply pages/common/u3d.md]: Ok, that makes sense. And the next example covers the case of version being a variable parameter. Thanks!

**Review by owenvoke — APPROVED** [review]

**Pull request merged**